batch/sql/estimated-current.sql

```diff
@@ -883,74 +825,25 @@ BEGIN
   SET cur_billing_date = CAST(UTC_DATE() AS DATE);
 
   IF msec_diff_rollup != 0 THEN
-    INSERT INTO aggregated_billing_project_user_resources_v2 (billing_project, user, resource_id, token, `usage`)
+    INSERT INTO aggregated_billing_project_user_resources_v3 (billing_project, user, resource_id, token, `usage`)
```

Comment:
And just checking here to make absolutely sure, `resource_id` and `deduped_resource_id` are the same in this table?

Reply:
In the billing tables, there is no deduped_resource_id. The v2 table is essentially the "resource_id" in attempt_resources and the v3 table "resource_id" is equivalent to the "deduped_resource_id" in the attempt resources table.

Reply:
Awesome, thanks!